wandb/sdk/lib/printer.py

```diff
@@ -64,6 +64,14 @@
 """Color from the 256-color palette for the progress symbol."""
 
 
+def new_printer() -> Printer:
+    """Returns a new printer based on the environment we're in."""
+    if ipython.in_jupyter():
+        return _PrinterJupyter()
+    else:
+        return _PrinterTerm()
```

Comment:
```suggestion
     return _PrinterTerm()
```
nit

Reply:
can do, but why?

Reply:
just i think it is better style in python if you have already a return you don't need the else for the other return

Reply:
ok, tried to google it, because i know that would be the case in python but looks like the opinions are varied

Reply:
they have slightly different meanings, by convention

```python
# If statements are edge cases, final return is the normal path:
if cond:
    return edge_case
return default


# All cases are equal, no case is preferred:
if cond:
    return cond_result
else:
    return other_result
```

Reply:
yeah, i just know the first is more idiomatic to python, but i don't have any preference, so if you feel the current is better that's ok :)